pkgs/top-level/all-packages.nix

```diff
@@ -2062,7 +2062,7 @@ in
 
   horst = callPackage ../tools/networking/horst { };
 
-  host = callPackage ../tools/networking/host { };
+  host = bind.host;
```

Comment:
Is a separate host package really required or could this package be deprecated in favor of dnsutils?